pkg/authorization/registry/clusterrole/proxy/proxy.go

```diff
@@ -1,124 +1,150 @@
 package proxy
 
 import (
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
 	metainternal "k8s.io/apimachinery/pkg/apis/meta/internalversion"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/runtime"
 	apirequest "k8s.io/apiserver/pkg/endpoints/request"
 	"k8s.io/apiserver/pkg/registry/rest"
+	"k8s.io/kubernetes/pkg/api"
+	"k8s.io/kubernetes/pkg/apis/rbac"
+	"k8s.io/kubernetes/pkg/client/clientset_generated/internalclientset/typed/rbac/internalversion"
 
-	authorizationapi "github.com/openshift/origin/pkg/authorization/apis/authorization"
-	clusterpolicyregistry "github.com/openshift/origin/pkg/authorization/registry/clusterpolicy"
-	"github.com/openshift/origin/pkg/authorization/registry/clusterrole"
-	roleregistry "github.com/openshift/origin/pkg/authorization/registry/role"
-	rolestorage "github.com/openshift/origin/pkg/authorization/registry/role/policybased"
-	"github.com/openshift/origin/pkg/authorization/rulevalidation"
+	authzapi "github.com/openshift/origin/pkg/authorization/apis/authorization"
```

Comment:
I changed it initally to make sure I was catching everything (compile error if not)
I can put it back, but man you guys love LOOOOOOOONG lines ...